src/PKSim.Presentation/Presenters/ContextMenus/MultipleSensitivityAnalysesContextMenu.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using OSPSuite.Assets;
+using OSPSuite.Core.Domain;
+using OSPSuite.Core.Domain.SensitivityAnalyses;
+using OSPSuite.Presentation.Core;
+using OSPSuite.Presentation.MenuAndBars;
+using OSPSuite.Presentation.Nodes;
+using OSPSuite.Presentation.Presenters;
+using OSPSuite.Presentation.Presenters.ContextMenus;
+using OSPSuite.Presentation.UICommands;
+using OSPSuite.Utility.Extensions;
+using IContainer = OSPSuite.Utility.Container.IContainer;
+
+namespace PKSim.Presentation.Presenters.ContextMenus
+{
+   public class MultipleSensitivityAnalysesContextMenuFactory : MultipleNodeContextMenuFactory<ClassifiableSensitivityAnalysis>
+   {
+      private readonly IContainer _container;
+
+      public MultipleSensitivityAnalysesContextMenuFactory(IContainer container)
+      {
+         _container = container;
+      }
+
+      protected override IContextMenu CreateFor(IReadOnlyList<ClassifiableSensitivityAnalysis> nodes, IPresenterWithContextMenu<IReadOnlyList<ITreeNode>> presenter)
+      {
+         var sensitivityAnalysisList = nodes.Select(pi => pi.SensitivityAnalysis).ToList().AsReadOnly();
```

Comment:
AsReadOnly? What is this?

Reply:
List implements IREadOnlyList anyways. so why do we need the two? 🤔

Reply:
Removed.